### PR TITLE
LVUIDF-21: added devise user

### PR DESCRIPTION
**Pull request by irynar**
Added user model through devise

To review:
- [x] @rtriska 
- [ ] @vyshnevska 

Optional:
- [ ] @yurko 

To merge:
- [x] @izagor

### CODE REVIEW

File: app/views/layouts/application.html.haml [hunk 1]
```diff
@@ -7,4 +7,7 @@
     = javascript_include_tag 'application'
     = csrf_meta_tags
   %body
-    = yield
+    .container
+      %p.notice= notice
+      %p.alert= alert
```
Comment (forecome) [inline app/views/layouts/application.html.haml]: What if no flash messages are present? DOM will have empty flash containers, which, probably, can impact on markup.

File: config/initializers/devise.rb [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+  # confirmation, reset password and unlock tokens in the database.
+  # Devise will use the `secret_key_base` on Rails 4+ applications as its `secret_key`
+  # by default. You can change it below and use your own secret key.
+  # config.secret_key = '0b1fb3820dfbf2513b58b873df16f56e594f420e3554bbfa54b4ae99e1f81f9023714d4fd69a31c28a43d09fe110b3e75d65ca8db8e5e8e7898156a59ca9da42'
+
+  # ==> Mailer Configuration
+  # Configure the e-mail address which will be shown in Devise::Mailer,
+  # note that it will be overwritten if you use your own mailer class
+  # with default "from" parameter.
+  config.mailer_sender = 'please-change-me-at-config-initializers-devise@example.com'
```
Comment (forecome) [inline config/initializers/devise.rb]: You forgot to change this line

File: db/migrate/20150709194238_devise_create_users.rb [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      t.datetime :remember_created_at
+
+      ## Trackable
+      t.integer  :sign_in_count, default: 0, null: false
+      t.datetime :current_sign_in_at
+      t.datetime :last_sign_in_at
+      t.inet     :current_sign_in_ip
+      t.inet     :last_sign_in_ip
+
+      ## Confirmable
```
Comment (forecome) [inline db/migrate/20150709194238_devise_create_users.rb]: It's not a notice, just a small remark for your information.
Those commented by default devise attributes are great to uncomment. Currently you not need them, but in future you probably will need to use confirmable of lockable functionality and you will need to create new migration, copy those lines into it and so on. You may also enable those functionalities in user model, but forget to create such migration and will have broken code.
But if you will create all needed (but even not used currently) fields right now, you will prevent all probable problems and free yourself or other people from making extra work.

**Comment by rtriska** [comment]
Need rebasing

**Comment by irynar** [comment]
Updated. Can you check and merge, please?

**Comment by rtriska** [comment]
again needs rebasing :man: 
And also I'm wondering about flash messages container. I'll create separate task for it. We should move this container under React control.

**Comment by irynar** [comment]
Rebase is done. Please merge/

**Comment by rtriska** [comment]
@izagor please merge. Code looks good for me

**Pull request merged**